modules/retail/pages/uyuni-retail-overview.adoc

```diff
@@ -0,0 +1,87 @@
+ifndef::backend-pdf[]
+[[uyuni-retail-overview]]
+= Retail Guide
+
+ifeval::[{mlm-content} == true]
+:noindex:
+endif::[]
+
+// HTML Publication date 
+**Publication Date:** {docdate}
+
+== Preface
+
+{productname} {smr} {productnumber} is an **open-source infrastructure management solution**, specifically **optimized for the retail industry**. 
+Built on the same technology as {productname}, it has been tailored to meet the unique operational needs of retail organizations.  
+
+Designed for **retail environments**, {productname} {smr} supports **point-of-service (POS) terminals**, allowing businesses to manage transactions, promotions, and customer loyalty programs. 
+Beyond retail, it can also be used in **educational institutions** to maintain student computers, as well as in **banks and hospitals** for self-service kiosks.  
+
+{productname} {smr} is suited for deployments that include **servers, workstations, point-of-service terminals, and other connected devices**. 
+It enables administrators to **install, configure, and update software across their infrastructure**, while streamlining **deployment and provisioning** of POS machines.  
+
+This guide provides a **comprehensive overview** of {productname} {smr}, covering its **initial installation and setup**.  
+
+ifeval::[{mlm-content} == true]  
```

Comment:
just remove these two ifevals. No reason to keep this here. We provide this info on the root/index page.